modules/fieldlinessequence/rendering/renderablefieldlinessequencesetup.cpp

```diff
@@ -0,0 +1,874 @@
+ * OpenSpace                                                                             *
+ *                                                                                       *
+ * Copyright (c) 2014-2017                                                               *
+ *                                                                                       *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this  *
+ * software and associated documentation files (the "Software"), to deal in the Software *
+ * without restriction, including without limitation the rights to use, copy, modify,    *
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software, and to    *
+ * permit persons to whom the Software is furnished to do so, subject to the following   *
+ * conditions:                                                                           *
+ *                                                                                       *
+ * The above copyright notice and this permission notice shall be included in all copies *
+ * or substantial portions of the Software.                                              *
+ *                                                                                       *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,   *
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A         *
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT    *
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF  *
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  *
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
+ ****************************************************************************************/
+
+#include <modules/fieldlinessequence/rendering/renderablefieldlinessequence.h>
+
+#ifdef OPENSPACE_MODULE_KAMELEON_ENABLED
+    #include <ccmc/Kameleon.h>
+    #include <modules/kameleon/include/kameleonhelper.h>
+#endif // OPENSPACE_MODULE_KAMELEON_ENABLED
+
+#include <openspace/engine/openspaceengine.h>
+#include <openspace/interaction/navigationhandler.h>
+#include <openspace/scene/scene.h>
+#include <openspace/scene/scenegraphnode.h>
+#include <openspace/util/timemanager.h>
+
+#include <ghoul/filesystem/filesystem.h>
+#include <ghoul/opengl/programobject.h>
+
+#include <fstream>
+#include <sstream>
+
+namespace {
+    std::string _loggerCat = "RenderableFieldlinesSequence";
+
+    // ----- KEYS POSSIBLE IN MODFILE. EXPECTED DATA TYPE OF VALUE IN [BRACKETS]  ----- //
+    // ---------------------------- MANDATORY MODFILE KEYS ---------------------------- //
+    const char* KEY_INPUT_FILE_TYPE         = "InputFileType";   // [STRING] "cdf", "json" or "osfls"
```

Comment:
Typically we use CamelCaseWithCapitalFirstLetter for these types of constants

Reply:
Thumbs up!